test_format_parents_verbose fails due to hardcoded relative date

We are getting `tests/test_chronos_serviceinit.py:290: AssertionError`. This is due to the test assertion checking that 2007-04-05T14:30 was 8 years ago. Due to how the relative time is being calculated, this is now 9 years ago.

```
>           assert "  Last Run: %s (2007-04-05T14:30, 8 years ago)" % PaastaColors.green("OK") in actual
E           assert ('  Last Run: %s (2007-04-05T14:30, 8 years ago)' % '\x1b[32mOK\x1b[0m') in '\n    - testservice testinstance\n      Last Run: \x1b[32mOK\x1b[0m (2007-04-05T14:30, 9 years ago)'
E            +  where '\x1b[32mOK\x1b[0m' = <function green at 0x3518d70>('OK')
E            +    where <function green at 0x3518d70> = PaastaColors.green
```

--- a/tests/test_chronos_serviceinit.py
+++ b/tests/test_chronos_serviceinit.py
@@ -13,7 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import contextlib
+import datetime
 
+import dateutil.parser
+import dateutil.relativedelta
 import mock
 import pytest
 
@@ -270,7 +273,8 @@ def test_format_parents_verbose():
         'name': 'myexamplejob',
         'parents': ['testservice testinstance']
     }
-    example_status = ('2007-04-05T14:30', chronos_tools.LastRunState.Success)
+    fake_last_datetime = '2007-04-01T17:52:58.908Z'
+    example_status = (fake_last_datetime, chronos_tools.LastRunState.Success)
     with contextlib.nested(
         mock.patch(
             'paasta_tools.chronos_tools.get_job_for_service_instance',
@@ -285,9 +289,13 @@ def test_format_parents_verbose():
             return_value=example_status
         ),
     ):
+        expected_years = dateutil.relativedelta.relativedelta(
+            datetime.datetime.now(dateutil.tz.tzutc()),
+            dateutil.parser.parse(fake_last_datetime)
+        ).years
         actual = chronos_serviceinit._format_parents_verbose(example_job)
         assert "testservice testinstance" in actual
-        assert "  Last Run: %s (2007-04-05T14:30, 8 years ago)" % PaastaColors.green("OK") in actual
+        assert "  Last Run: %s (2007-04-01T17:52, %s years ago)" % (PaastaColors.green("OK"), expected_years) in actual
 
 
 def test_format_schedule_dependent_job():

